app/core/configurator.py

```diff
@@ -0,0 +1,44 @@
+"""
+Constants for submission details
+"""
+
+DRAFT_MODE_DETAILS = """In draft mode, submissions may be incomplete."""
+
+NAME_DETAILS = """Please provide an informal but informative name of the method used.
+The name must not exceed 40 characters."""
+
+COMPUTE_TIME_DETAILS = """Please provide the average compute time across all of the molecules.
+For physical methods, report the GPU and/or CPU compute time in hours.
+For empirical methods, report the query time in hours.
+Create a new line for each processor type."""
+
+COMPUTING_AND_HARDWARE_DETAILS = """Please provide details of the computing resources that were used to train models and make predictions.
+Please specify compute time for training models and querying separately for empirical prediction methods.
+Provide a detailed description of the hardware used to run the simulations."""
+
+SOFTWARE_DETAILS = """List all major software packages used and their versions.
+Create a new line for each software."""
+
+CATEGORY_OPTIONS = ("Physical (MM)", "Physical (QM)", "Empirical", "Mixed")
+CATEGORY_CHOICES = zip(CATEGORY_OPTIONS, CATEGORY_OPTIONS)
```

Comment:
What does this do?

Reply:
should be ```list(zip,``` thanks

Reply:
django choices have to be a list of tuples, in this case I want both elements of the tuple to be the same